src/applications/personalization/preferences/SetPreferences.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Link, withRouter } from 'react-router';
+import { connect } from 'react-redux';
+
+import { chunk, fill } from 'lodash';
+
+import { benefitChoices } from './helpers';
+
+import { setPreference, savePreferences, fetchPreferences } from './actions';
+
+import localStorage from '../../../platform/utilities/storage/localStorage';
+
+import Checkbox from '../../gi/components/Checkbox.jsx';
+
+class SetPreferences extends React.Component {
+  componentDidMount() {
+    this.props.fetchPreferences();
+    localStorage.setItem('dashboardLastVisitedAt', Date.now());
+  }
+
+  handleSave = () => {
+    this.props.savePreferences(this.props.preferences.dashboard);
+    this.props.router.push('/');
+  }
+
+  handlePreferenceToggle = (slug) => {
+    return () => {
+      this.props.setPreference(slug, !this.props.preferences.dashboard[slug]);
+    };
+  }
+
+  render() {
+    return (
+      <div className="row user-profile-row">
+        <div className="small-12 columns">
+          <h1 id="dashboard-title">Find VA Benefits</h1>
+          <p className="va-introtext">Tell us which benefits you're interested in, so we can help you apply. Select one or more of the types of benefits below, and we'll help you get started.</p>
+          <div>
+            {chunk(benefitChoices, 3).map((c, i) => {
+              return (
+                <div key={i} className="preferences-row">
+                  {c.concat(fill(Array(3 - c.length), null)).map((e, ix) => {
+                    let itemContent;
```

Comment:
This is a little confusing to me with the `chunk`, `fill` (not familiar with those Lodash functions), and the array functions. Might help clarify things with some local variables.

Reply:
This code is a bit arcane, could be clarified via comments or variable names

Reply:
This logic has been removed as we can simply set the width and the existing flex-based styles will achieve this layout.